Small improvements and solved bugs in the volto-slate package (#7179)
plone/volto: packages/volto-slate/README.md

This is a Volto core add-on since Volto 16.0.0.
It is released within Volto in the same code base.

For projects using either the Volto 16.x.x or 17.x.x series, it is _not_ required to be installed as an add-on.

From Volto 18.0.0 onwards (including alphas), it is a released package as a stand-alone add-on in Volto.
It is declared as part of its `dependencies`.

> See [`volto-slate`](https://6.docs.plone.org/volto/configuration/volto-slate/) for more information about Volto Slate and how to use it.

Reviewer: ```suggestion
See the documentation for [`volto-slate`](https://6.docs.plone.org/volto/configuration/volto-slate/) for more information.
```
